lib/ohai/plugins/darwin/cpu.rb

```diff
@@ -22,10 +22,12 @@
 
   collect_data(:darwin) do
     cpu Mash.new
-    so = shell_out("sysctl -n hw.physicalcpu")
+    so = shell_out("sysctl -n hw.packages")
     cpu[:real] = so.stdout.to_i
-    so = shell_out("sysctl -n hw.logicalcpu")
+    so = shell_out("sysctl -n hw.physicalcpu")
     cpu[:total] = so.stdout.to_i
+    so = shell_out("sysctl -n hw.logicalcpu")
+    cpu[:cores] = so.stdout.to_i
```

Comment:
checked the darwin APIs and they're the way i assumed they be from the names...  so the only remaining question is if our ohai APIs are backwards from what i think they should mean...